tools/acorn-optimizer.js

```diff
@@ -287,13 +299,13 @@ function hasSideEffects(node) {
 // as they appear (like ArrowFunctionExpression). Instead, we do a conservative
 // analysis here.
 
-function JSDCE(ast, aggressive) {
+function runJSDCE(ast, aggressive) {
```

Comment:
Does eslint rename functions?

Reply:
There is a rule that says that any function that starts with a Capital letter must be a constructor function (and this isn't one).